recipe/meta.yaml

```diff
@@ -7,16 +7,19 @@ package:
 source:
   fn: ncurses-{{ version }}.tar.gz
   url: http://ftp.gnu.org/pub/gnu/ncurses/ncurses-{{ version }}.tar.gz
-  md5: 8cb9c412e5f2d96bc6f459aa8c6282a1
+  md5: ee13d052e1ead260d7c28071f46eefb1
   patches:
     # http://lists.gnu.org/archive/html/bug-ncurses/2011-04/txtkWQqiQvcZe.txt
     - fix.patch
     # http://lists.gnu.org/archive/html/bug-ncurses/2015-01/msg00016.html
     - ncurses-5.9-gcc-5.patch
+    - clang.patch
+    # This patch is not needed with GCC 4.8.5
+    #- cxx1z-remove-dynamic-exception-specifications.patch
```

Comment:
Should we be adding this patch if we are not using it or should we add it when we switch compilers?